src/service/jwt.service.js

```diff
@@ -0,0 +1,46 @@
+import 'dotenv/config';
+import jwt from 'jsonwebtoken';
+import { ConsoleLoger } from '../untils/consoleLoger.js';
```

Comment:
There is a typo in 'ConsoleLoger'. It should be 'ConsoleLogger'. Ensure that the import statement and all usages are corrected.